tools/ansible/tasks/logging/ReadMe.md

```diff
@@ -0,0 +1,34 @@
+# Logging on Windows instance
+
+These playbooks install Fluentd on Windows Instance, in order to enable logging support.
+
+Before running,
+- Set up an OpenShift cluster.
+- Set up a Windows instance as part of the cluster. You can use `wni` tool to set it up by following this [link](https://github.com/openshift/windows-machine-config-operator/tree/master/tools/windows-node-installer).
+- Ensure Ansible environment is set up to run the playbook.
+- Ensure a valid inventory file is set up.
+
+Run `main.yml` with the following command:
+```
+$ ansible-playbook -v -i <inventory_file> main.yml --extra-vars "ruby_fetch_url=<ruby_fetch_url>"
+```
+As a part of Fluentd installation, Ruby DevKit is required.
```

Comment:
Shouldn't this be < 2.6?
 cc @sebsoto?

Reply:
I'm not sure, I only tested on 2.4.6

Reply:
According to https://rubyinstaller.org/add-ons/devkit.html it should be >=2.4.
Fluentd requires MSYS2, however I haven't tested with versions below 2.4 (they are bundled with MSYS1)